db/schema.rb

```diff
@@ -46,11 +48,6 @@
     t.index ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true
   end
 
-  create_table "place_statuses", force: :cascade do |t|
```

Comment:
was this deletion intended?

Reply:
no, it was deleted automatically during migrations

Reply:
yes, this table was deleted in previous PR